pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -1171,7 +1174,63 @@ def showAxis(self, axis, show=True):
     def hideAxis(self, axis):
         """Hide one of the PlotItem's axes. ('left', 'bottom', 'right', or 'top')"""
         self.showAxis(axis, False)
-            
+        
+    def showAxes(self, selection, showValues=True, size=False):
+        """ 
+        Convenience method for quickly configuring axis settings.
+        
+        Parameters
+        ----------
+        selection: boolean or tuple of booleans (left, top, right, bottom)
+            Determines which AxisItems will be displayed.
+            A single boolean value will set all axes, 
+            so that ``showAxes(True)`` configures the axes to draw a frame.
+        showValues: optional, boolean or tuple of booleans (left, top, right, bottom)
+            Determines if values will be displayed for the ticks of each axis.
+            True value shows values for left and bottom axis (default).
+            False shows no values.
+            None leaves settings unchanged.
+            If not specified, left and bottom axes will be drawn with values.
+        size: optional, float or tuple of floats (width, height)
+            Reserves as fixed amount of space (width for vertical axis, height for horizontal axis)
+            for each axis where tick values are enabled. If only a single float value is given, it
+            will be applied for both width and height. If `None` is given instead of a float value,
+            the axis reverts to automatic allocation of space.
+        """
+        if selection is True: # shortcut: enable all axes, creating a frame
+            selection = (True, True, True, True)
+        elif selection is False: # shortcut: disable all axes
+            selection = (False, False, False, False)
+        if showValues is True: # shortcut: defaults arrangement with labels at left and bottom
+            showValues = (True, False, False, True)
+        elif showValues is False: # shortcut: disable all labels
+            showValues = (False, False, False, False)
+        elif showValues is None: # leave labelling untouched
+            showValues = (None, None, None, None)
+        if size is not False and not isinstance(size, collections.abc.Sized):
+            size = (size, size) # 
```

Comment:
a `#` managed to escape the mass deletion of comments!

Reply:
It appears that this dangerous specimen has devoured the entire team of commentary it had been assigned to protect. Horrible thing, that. I have replaced the `#` with a less dangerous one, and recruited a new volunteer comment to explain that this is intended to `# make sure that size is either False or a full set of (width, height)`

Reply:
😆